src/app/zap-templates/templates/app/att-storage.zapt

```diff
@@ -13,11 +13,11 @@
 #define CLUSTER_MASK_DEFAULT_RESPONSE_FUNCTION (0x04)
 // Does this cluster have message sent function?
 #define CLUSTER_MASK_MESSAGE_SENT_FUNCTION (0x08)
```

Comment:
Please file a followup to get rid of all the unused bits here; we seem to have several.

Reply:
done
https://github.com/project-chip/connectedhomeip/issues/24984